src/app/view-modes/collation/collation.component.html

```diff
@@ -1,6 +1,9 @@
 <gridster [options]="options">
     <gridster-item [item]="textPanelItem">
-        <evt-text-panel></evt-text-panel>
+        <evt-text-panel
+            [editionLevelID]="currentEditionLevel | async"
+            (editionLevelChange)="handleEditionLevelChange($event)"
+        ></evt-text-panel>
```

Comment:
Is this selector necessary in the collation view mode? I think it's not necessary in any case

Reply:
Ok. Maybe we should just set the param directly as "critical" in this case, what do you think?